### PR TITLE
Create new user via Systems -> Users screen

### PR DESCRIPTION
**Pull request by pqr**
This is implementation for issue #1527

Add ability to create new user via Users admin screen.

### CODE REVIEW

File: stubs/app/Orchid/Screens/User/UserListScreen.php [hunk 1]
```diff
@@ -57,7 +58,11 @@ public function query(): array
      */
     public function commandBar(): array
     {
-        return [];
+        return [
+            Link::make(__('Add'))
+                ->icon('plus')
+                ->href(route('platform.systems.users.create')),
```
Comment (tabuna) [inline stubs/app/Orchid/Screens/User/UserListScreen.php:64]: We can use a shorter notation:


```php
Link::make(__('Add'))
    ->icon('plus')
    ->route('platform.systems.users.create'),
```

File: stubs/app/Orchid/Screens/User/UserEditScreen.php [hunk 2]
```diff
@@ -150,14 +162,23 @@ public function save(User $user, Request $request)
             ->collapse()
             ->toArray();
 
+        $userData = $request->get('user');
```
Comment (tabuna) [inline stubs/app/Orchid/Screens/User/UserEditScreen.php:165]: It looks like the $ userData variable is not being used. I think you meant it?

```php
$password = $request->input('user.password');

if (!$user->exists && !empty($password)) {
    $user->password = Hash::make($password);
}
```

Comment (pqr) [reply stubs/app/Orchid/Screens/User/UserEditScreen.php:165]: My fail with last minute changes.
Fixed

File: stubs/app/Orchid/Screens/User/UserListScreen.php (hunk 1 above)
Comment (pqr) [reply stubs/app/Orchid/Screens/User/UserListScreen.php]: Nice addition, thanks!

**Pull request merged**